src/google-maps/google-map/google-map.ts

```diff
@@ -172,6 +175,10 @@ export class GoogleMap implements OnChanges, OnInit, OnDestroy {
    */
   @Output() zoomChanged = new EventEmitter<void>();
 
+  @ContentChildren(GoogleMapMarker) set markers(markers: QueryList<GoogleMapMarker>) {
+    this._markers.next(markers.toArray());
```

Comment:
Angular will only set `@ContentChildren` once, but the querylist's `changes` property emits whenever the markers change; you would want to be listening to changes on that

Reply:
Are you sure it only sets it once? I've experimented with having the child components change through click events and and this code successfully adds and removes the markers, so it seems to set it whenever the content changes. The documentation doesn't clarify this point, although it does recommend using QueryList in the way you suggested. I will make the change you suggested, but I wanted to check with you first if the current implementation had problems that I wasn't seeing.

Reply:
I checked w/ Kara and yeah, it does actually get set every time, which surprised me. I would still go with the `changes` stream, though, since that's what we do everywhere else.

Reply:
Yeah, upon further discussion, Kara says that the fact the setter gets called multiple times is an implementation detail and that people shouldn't _depend_ on it since it might change  someday.

Reply:
Good to know. Done.